### PR TITLE
Task 2 - Cargo class constructors & getters

### LINKED ISSUE #2: Task 2 - Cargo (gettery)
Do klasy Cargo dopisz gettery oraz odpowiedni konstruktor, który wypełni wszystkie pola tej klasy.

### PR DESCRIPTION
**Pull request by mariuszlisowski**

### CODE REVIEW

File: shm/cargo.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Cargo& Cargo::operator+=(size_t amount) {
+    amount_ += amount;
+    return *this;
+}
+Cargo& Cargo::operator-=(size_t amount) {
+    amount_ -= amount;
+    return *this;
+}
+
+std::string Cargo::getName() { return name_; }
```
Comment (Petrus9107) [inline shm/cargo.cpp]: best practise is to use "const getters" -> std::string Cargo::getName() **const** { return name_;), same to the rest of getters

File: shm/cargo.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+size_t Cargo::getAmount() { return amount_; }
+size_t Cargo::getBasePrice() { return basePrice_; }
+
+void Cargo::setName(std::string name) { name_ = name; }
```
Comment (Petrus9107) [inline shm/cargo.cpp]: should pass setters arguments by reference

File: shm/cargo.cpp (hunk 1 above)
Comment (mariuszlisowski) [reply shm/cargo.cpp]: That's right!

File: shm/cargo.cpp (hunk 2 above)
Comment (m-kop) [reply shm/cargo.cpp]: please add const &

File: shm/cargo.hpp [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <string>
+
+class Cargo {
+public:
+    Cargo();
```
Comment (m-kop) [inline shm/cargo.hpp:7]: is there a use case for default constructor? I'm asking as I cannot figure it out

**Review by m-kop — CHANGES_REQUESTED** [review]

**Review by m-kop — APPROVED** [review]

**Pull request merged**